doc/cygnus-ngsi/installation_and_administration_guide/backends_as_sth.md

```diff
@@ -0,0 +1,40 @@
+
+##<a name="top"></a>How it works
+There are two approaches when deciding which data must be removed from existent historics:
+
+* By <b>capping</b> data "records"(*) once certain size limit has been reached. In other words, to control only the last N records are stored.
+* By <b>expirating</b> "records"(*) once certain keepalive limit has been reached. In other wods, to control only records added in the last N seconds are stored.
+
+The above can be controlled by means of specific configuration parameters in certain sinks:
+
+| Parameter | Mandatory | Default value | Comments |
+|---|---|---|---|
+| persistence\_policy.max_records | no | -1 | Maximum number of records allowed for a table before it is capped. `-1` disables this policy. |
+| persistence\_policy.expiration_time | no | -1 | Maximum number of seconds a record is maintained in a table before expiration. `-1` disables this policy. |
+| persistence\_policy.checking_time | no | 3600 | Frequency (in seconds) at which the sink checks for record expiration. |
+
+Which sinks provide this kind of functionality? For the time being:
+
+* `NGSIMySQLSink`
+* `NGSICKANSink`
+
+(*) A record may mean many different things depending on the persistence backend: a Json entry in a HDFS file, a row in a MySQL table, a record in a CKAN resoeurce...
+
+[Top](#top)
+
+##<a name="top"></a>The special case of `NGSIMongoSink` and `NGSISTHSink`
+`NGSIMongoSink` and `NGSISTHSink` already implemented this kind of functionality, since the data stored in MongoDB and STH Comet was wanted to be a short-term historic from the very begining. In these cases, the parameters controlling the functionality are:
+
+| Parameter | Mandatory | Default value | Comments |
+|---|---|---|---|
+| data\_expiration | no | 0 | Collections will be removed if older than the value specified in seconds. The reference of time is the one stored in the `recvTime` property. Set to 0 if not wanting this policy. |
+| collections\_size | no | 0 | The oldest data (according to insertion time) will be removed if the size of the data collection gets bigger than the value specified in bytes. Notice that the size-based truncation policy takes precedence over the time-based one. Set to 0 if not wanting this policy. Minimum value (different than 0) is 4096 bytes. <b>Only available for `NGSIMongoSink`</b>. |
+| max\_documents | no | 0 | The oldest data (according to insertion time) will be removed if the number of documents in the data collections goes beyond the specified value. Set to 0 if not wanting this policy. <b>Only available for `NGSIMongoSink`</b>. |
+
+There are also differences in the implementations: while MongoDB natively provides mechanisms for controlling the collections growing at collection creation time -either by size either by time-, MySQL and CKAN don't provide something similar. In these cases, the functionality has been built on top of the API through delete operations calculated by Cygnus itself.
```

Comment:
I know the MongoDB native mechanisms for controlling collections growing by size (they call that "cappend collections"). However, I didn't not aoubt time-based control. Could you elaborate?

(This is just curiosity about MongoDB tech :)